data/elasticsearch/elasticsearch.go

```diff
@@ -23,26 +24,67 @@ func (es *ElasticSearch) Connect(url string) error {
 	return err
 }
 
-// Index stores a document inside elastic search
-func (es *ElasticSearch) Index(esIndex string, esType string, body interface{}) error {
+// CreateIndexIfNotExists Creates an index if it doesn't already exists
```

Comment:
We are going to use this package for fetching data only (to the best of my knowledge). The log worker should be responsible for ensuring the index exists for the data it will be writing. Are you aware of any other use cases for elasticsearch?